assets/css/preview.css

```diff
@@ -14,6 +14,28 @@
 	clear: both;
 }
 
+.apple-news-settings-left .collapsed td {
+	display: none;
+	visibility: hidden;
+}
+
+.apple-news-settings-left th:before {
+	background: #333;
+	border-radius: 1em;
+	color: #eee;
+	content: '-';
+	display: inline-block;
+	height: 1.5em;
+	line-height: 1.25em;
+	margin-right: 0.5em;
+	text-align: center;
+	width: 1.5em;
+}
+
+.apple-news-settings-left .collapsed th:before {
+	content: '+';
```

Comment:
`content: "\271A";` if we decide to go with Hex instead of the straight-up symbol!

Reply:
Good to know! Is there a particular reason why we would want to use hex vs. the symbol? I have no strong feelings either way.

Reply:
Consistency basically... but as far as I can see this is the only symbol we're using for this right now. If we end up using more esoteric symbols that aren't easy to type, hex can be more useful.